src/components/BubbleBox/BubbleBox.js

```diff
@@ -1,22 +1,33 @@
-import React, { useContext } from 'react'
-import './_BubbleBox.scss';
-import BigBubble from '../BigBubble/BigBubble';
-import {DataContext} from '../../contexts/DataContext'
-  
+import React, { useContext } from "react";
+import "./_BubbleBox.scss";
+import BigBubble from "../BigBubble/BigBubble";
+import { DataContext } from "../../contexts/DataContext";
+import SmallBubble from "../SmallBubble/SmallBubble";
+import { nanoid } from "nanoid";
+
+const BubbleBox = () => {
+  const data = useContext(DataContext);
+  console.log("this is data", data);
+  const relatedItems = data.data.relatedItems;
+  const createBubbles = () => {
+    return relatedItems.map((item, index) => {
+      item.id = `bubble${index+=1}`;
```

Comment:
How is this `id` value utilized in the SmallBubble component? Is it necessary to target the element for something? I just want to make sure you're adding the `id` for a specific reason and not because you like you just need to have it.

Reply:
Hi Robbie! Thanks for looking this over -- we are giving some specific positioning to our small bubbles to give them a cloud-formation on the main page and are utilizing those ids to provide the specific positioning to each bubble in our CSS (left & top).

Reply:
Got it - sounds good. In that case, `id` usually means something very specific - a unique identifier that stays the same for a given piece of data. Maybe this data you're using could have a different name that is tied closer to it's purpose of positioning.